### PR TITLE
Final design refinements

### PR DESCRIPTION
**Pull request by katsiarynalashcheuskaya**
## Summary:

View the entire App and correct styles where necessary.

## Requirements:
- Update styles on the quiz page. 
- Fix node.js image.
- Add missing snackbars. 
- Other minor styling changes.

### CODE REVIEW

File: src/pages/Main/QuizCard.jsx [hunk 1]
```diff
@@ -189,19 +189,20 @@ const QuizCard = ({
                   sx={{
                     marginTop: '10px',
                     color: customColors.greyDark,
-                    fontWeight: 'bold',
                   }}
                 >
-                  {bestScore && `Best Score: ${bestScore}%`}
+                  {bestScore ? `Score: ${bestScore}%` : ''}
```
Comment (ezbc) [inline src/pages/Main/QuizCard.jsx:194]: Good catch!

File: src/pages/Main/Quizzes.jsx [hunk 2]
```diff
@@ -29,11 +34,21 @@ export const Quizzes = ({ changeFilter, activeFilters, searchValue }) => {
...
+        setSnackbar({
+          isOpened: true,
+          severity: 'success',
+          message: 'Quiz added to favorites.',
+        });
       } catch (error) {
-        console.error('Error adding to favorites:', error);
+        setSnackbar({
+          isOpened: true,
+          severity: 'error',
```
Comment (ezbc) [inline src/pages/Main/Quizzes.jsx:45]: These severities would be easiest to maintain as constants. It's easy to misspell a string and cause an error. If we misspell a constant our IDE will tell us immediately. The constants could look like :

```jsx
severities.ERROR
```

File: src/pages/SignUp.jsx [hunk 3]
```diff
@@ -186,27 +199,38 @@ export default function SignUp() {
...
-                          aria-label="toggle password visibility"
-                          onClick={handleClickShowPassword}
-                          onMouseDown={handleMouseDownPassword}
-                        >
-                          {showPassword ? <VisibilityOff /> : <Visibility />}
-                        </IconButton>
-                      </InputAdornment>
-                    }
-                  />
+                  <FormControl fullWidth variant="outlined" margin="normal">
```
Comment (ezbc) [inline src/pages/SignUp.jsx:202]: This form control component looks duplicated with the one in Login.jsx. Can you think of a way to deduplicate?

**Review by ezbc — CHANGES_REQUESTED** [review]
This is valuable cleanup :100: and the snackbar is useful. I have just a couple refactor suggestions that are minor.

File: src/pages/Main/Quizzes.jsx (hunk 2 above)
Comment (katsiarynalashcheuskaya) [reply src/pages/Main/Quizzes.jsx:45]: Updated this in my next PR

File: src/pages/SignUp.jsx (hunk 3 above)
Comment (katsiarynalashcheuskaya) [reply src/pages/SignUp.jsx:202]: Refactored this in my next PR

**Review by in43sh — APPROVED** [review]

**Pull request merged**